src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -53,3 +54,153 @@ export function printTable( data: Object ) {
 
 	console.log( formattedData );
 }
+
+export function processComponentOptionInput( passedParam: string, type: string ) {
+	// cast to string
+	const param = passedParam + '';
+	if ( param.includes( '/' ) ) {
+		return {
+			mode: 'local',
+			dir: param,
+		};
+	}
+
+	if ( type === 'jetpack' && param === 'mu' ) {
+		return {
+			mode: 'inherit',
+		};
+	}
+
+	return {
+		mode: 'image',
+		image: DEV_ENVIRONMENT_CONTAINER_IMAGES[ type ].image,
+		tag: param,
+	};
+}
+
+type NewInstanceOptions = {
+	title: string,
+	multisite: boolean,
+	phpVersion: string,
+	wordpress: string,
+	muPlugins: string,
+	jetpack: string,
+	clientCode: string
+}
+
+const promptIntro = 'This is a wizard to help you set up you local dev environment.\n\n' +
+	'Sensible defaualt values were pre-selected for convinience. ' +
+	'You can also choose to create multiple different environments with different settings using the --slug option.\n\n';
```

Comment:
Not sure if we should move this into the constant files.

Reply:
Done